modular-code.md

```diff
@@ -11,36 +11,49 @@ Development does not want to spend the extra resources needed to develop modular
 * Time commitments might already have been made for customer deliveries (not changeable).
 
 **Forces:**
-* There is a learning curve to writing code that can be reused.
+* There is a learning curve to writing code that can be reused. Developers might not know how to write modular code. Education might be needed.
 * Extra documentation is required for reusable code.
 * Some companies have a common components group that develops reusable code, but others feel that such components should be developed by those business lines that are using the components and a library of common components could be established.
-* Developers might not know how to write modular code. Education might be needed. 
 * Might be a fear that if not done properly, quality might be impacted.
 * Developers might not have incentive to write modular code (due to their tight schedules and lack of a mandate).
 * You might be dealing with legacy systems (can't be simply refactored or rewritten).
 * Requirements might be different for writing modular code.
 * Architectural constraints might impact modularity.
 * Developers who develop monolithic code bases might lack the perspective of how modularity might improve how they work.
+* If there is frequent turnover of team members, modularization may not be a priority.
```

Comment:
I don't understand the connection between these ideas.  Modular code seems even _more_ useful with rotating team members so that no one has to keep the whole monolith in their head?

Reply:
Got it.

Reply:
I'd say that's less about turnover of team members than just under short-term time pressure in-general.  Could be for a variety of reasons but we'd expect to see that result no matter what outside force is driving that pressure.

Reply:
There are two points that address the feedback by Russ that are already in the pattern:
> * Developers might have no incentive to write modular code (due to their tight schedules and lack of a mandate).
> * If there is frequent turnover of team members, modularization may not be a priority.

I moved those points side-by-side now and will let this be worked out when the patten gets leveled up later (if it does).